packages/template-retail-react-app/app/components/drawer-menu/drawer-menu.jsx

```diff
@@ -399,7 +407,15 @@ DrawerMenu.propTypes = {
     /**
      * Function called when the drawer logo is clicked.
      */
-    onLogoClick: PropTypes.func
+    onLogoClick: PropTypes.func,
+    /**
+     * .
+     */
+    itemsKey: PropTypes.string,
+    /**
+     * Component to be rendered for each individual menu item.
+     */
+    itemComponent: PropTypes.elementType
 }
 
-export default DrawerMenu
+export {DrawerMenu}
```

Comment:
This is a breaking change for extensibility, right? 

Where shall we add documentation for migration? How about adding steps the changelog? I think it's important to capture the migration steps to provide an easy upgrade.